### PR TITLE
[tests-only][full-ci]Adjust Robots.txt for ocis

### PR DESCRIPTION
**Pull request by SagarGi**
### Description
This PR adjust to get the content from `robots.txt` file from the `ocis` server root. This issue https://github.com/owncloud/ocis/issues/1314 was closed but the code adjustment was not done yet.

### Related issue
https://github.com/owncloud/QA/issues/716

### CODE REVIEW

File: tests/acceptance/features/bootstrap/FeatureContext.php [hunk 1]
```diff
@@ -2813,6 +2813,23 @@ public function readFileInServerRoot(string $path):void {
...
+	/**
+	 * send request to read a server file for ocis
+	 *
+	 * @param string $path
+	 *
+	 * @return string
+	 */
+	public function readFileInServerRootForOCIS(string $path):string {
+		$pathToOcis = \getenv("PATH_TO_OCIS");
+		$targetFile = $pathToOcis . "/" . "services/web/assets" . "/" . $path;
```
Comment (SagarGi) [inline tests/acceptance/features/bootstrap/FeatureContext.php:2825]: not sure if this is the correct `robots.txt` path on ocis

File: tests/acceptance/features/bootstrap/FeatureContext.php [hunk 2]
```diff
@@ -2882,24 +2899,32 @@ public function theLocalStorageMountForIsRenamedTo(string $mount, string $target
...
+		if (OcisHelper::isTestingOnOcis()) {
+			$fileContent = $this->readFileInServerRootForOCIS($path);
+			Assert::assertSame(
+				$content,
+				$fileContent,
+				"The content of the file does not match with '$content'"
+			);
+		} else {
+			$this->readFileInServerRootForCore($path);
+			Assert::assertSame(
```
Comment (kiranparajuli589) [inline tests/acceptance/features/bootstrap/FeatureContext.php:2911]: can we use `$this->theHTTPStatusCodeShouldBe(200)` ?

File: tests/acceptance/features/bootstrap/FeatureContext.php [hunk 3]
```diff
@@ -2882,24 +2899,32 @@ public function theLocalStorageMountForIsRenamedTo(string $mount, string $target
...
+				200,
+				$this->getResponse()->getStatusCode(),
+				"Failed to read the file $path"
+			);
+			$fileContent = HttpRequestHelper::getResponseXml(
```
Comment (kiranparajuli589) [inline tests/acceptance/features/bootstrap/FeatureContext.php:2916]: there is already a `$this->getResponseXml()` fn to get xml response of last response

File: tests/acceptance/features/bootstrap/FeatureContext.php (hunk 1 above)
Comment (kiranparajuli589) [reply tests/acceptance/features/bootstrap/FeatureContext.php:2825]: I can see this file inside the mentioned path in my local setup.

File: tests/acceptance/features/bootstrap/FeatureContext.php [hunk 4]
```diff
@@ -2882,19 +2899,18 @@ public function theLocalStorageMountForIsRenamedTo(string $mount, string $target
...
-		$fileContent = (string) $fileContent->data->element->contentUrlEncoded;
-		$fileContent = \urldecode($fileContent);
-
+		if (OcisHelper::isTestingOnOcis()) {
+			$fileContent = $this->readFileInServerRootForOCIS($path);
+		} else {
+			$this->readFileInServerRootForCore($path);
+			$this->theHTTPStatusCodeShouldBe(200, 'Failed to read the file $path');
+			$fileContent = $this->getResponseXml(
+				$this->getResponse(),
```
Comment (kiranparajuli589) [inline tests/acceptance/features/bootstrap/FeatureContext.php:2908]: i think this param is also needed, the last response is automatically set if nothing is provided.

**Review by phil-davis — APPROVED** [review]

**Review by kiranparajuli589 — APPROVED** [review]

**Comment by phil-davis** [comment]
Stupid drone: https://drone.owncloud.com/owncloud/core/36447/172/12
`services aren't ready in 10m0s`

Restarted.

**Pull request merged**